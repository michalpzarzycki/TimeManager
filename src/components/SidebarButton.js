import React from 'react';
import styles from './SidebarButton.module.css'

export default function SidebarButton() {
    return <div className={styles.sidebarButtonContainer}>
        <div className={styles.sidebarButtonBox}>
            HELLO
        </div>
    </div>
}