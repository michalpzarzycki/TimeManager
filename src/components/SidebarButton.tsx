import React from 'react';
import styles from './SidebarButton.module.css'

export default function SidebarButton({toggleSidebar, toggle} : any) {
    return <div 
    className={styles.sidebarButtonContainer}
    onClick = {() => toggleSidebar()}>
        <div className={!toggle ? styles.sideBarButtonBoxOpen : styles.sideBarButtonBoxClosed}>
            
        </div>
    </div>
}