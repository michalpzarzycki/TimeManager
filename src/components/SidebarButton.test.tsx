import React from 'react'
import {shallow} from 'enzyme'
import SidebarButton from './SidebarButton';


test('Expext to render SidebarButton component', () => {
    expect(shallow(<SidebarButton />)).toMatchSnapshot()
})