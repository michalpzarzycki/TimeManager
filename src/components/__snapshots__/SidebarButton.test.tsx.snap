// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render SidebarButton component 1`] = `ShallowWrapper {}`;
